UnicornRender/video/include/unicorn/video/OrthographicCamera.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_ORTHOGRAPHICCAMERA_HPP
+#define UNICORN_VIDEO_ORTHOGRAPHICCAMERA_HPP
+
+#include <unicorn/utility/SharedMacros.hpp>
+#include <unicorn/video/CameraProjection.hpp>
+#include <unicorn/system/Window.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+/** @brief Orthographic camera projection controller */
+class OrthographicCamera final : public CameraProjection
+{
+public:
+    UNICORN_EXPORT OrthographicCamera(system::Window* window, glm::mat4& cameraProj);
+
+    UNICORN_EXPORT void UpdateProjection() override;
+
+    /** @brief Changes scale by adding scaleOffset to it */
+    UNICORN_EXPORT void Scale(float scaleOffset);
```

Comment:
Is this method really that necessary when you have getter and setter for the scale?
If it is `AddScale` or something with a verb would be a better name.